scripts/integration_runner/custom_modules/neat_integration_runner/auth/neat_cicd_testing.group.yaml

```diff
@@ -34,3 +34,16 @@ capabilities:
         - OWNER
       scope:
         all: {}
+  - rawAcl:
```

Comment:
This is how you extend the test runner to have more access. 

Then you run the command `make run-toolkit`